blueprints/component-test/index.js

```diff
@@ -22,13 +39,41 @@ module.exports = {
   locals: function(options) {
     var dasherizedModuleName = stringUtil.dasherize(options.entity.name);
     var componentPathName = dasherizedModuleName;
-    if(options.pod && options.path !== 'components' && options.path !== '') {
+    var testTypeDefinition = "integration: true";
+    var friendlyTestDescription = testInfo.description(options.entity.name, "Integration", "Component");
+    var testContent = "assert.expect(1);" + EOL + EOL +
+      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
+      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+      "  // Provide a template (string or precompiled) for this.render()" + EOL +
+      "  this.render();" + EOL + EOL +
```

Comment:
@stefanpenner https://github.com/babel/ember-cli-babel/pull/42 is currently blocking https://github.com/pangratz/ember-cli-htmlbars-inline-precompile/issues/7. When those are resolved, I will open a PR for using `htmlbars-inline-precompile` inside component integration tests, as suggested in https://github.com/pangratz/ember-cli-htmlbars-inline-precompile/issues/7#issuecomment-103967882.